service-api/src/main/java/greencity/service/ubs/NotificationService.java

```diff
@@ -233,4 +233,14 @@ PageableDto<NotificationShortDto> getAllNotificationsForUser(String userUuid,
      * @author Roman Kasarab
      */
     void deleteNotification(Long notificationId, String userUuid);
+
+    /**
+     * Notify user that order has unpaid status for a long time. This method is used
```

Comment:
what does `a long time` mean here?

Reply:
The Java doc was written by jetbrains ai😁, he did a bad job, and I changed the description myself